scripts/package.js

```diff
@@ -1,27 +1,10 @@
 const exec = require('child_process').exec
 const fs = require('fs')
-const fse = require('fs-extra')
 
-fs.rmSync('package', { recursive: true, force: true })
+fs.rmSync('lib', { recursive: true, force: true })
 
-exec('mkdir package && cd package && mkdir lib && babel --config-file ../package.babel.js ../src -d lib --extensions .js,.jsx,.snap --copy-files --no-copy-ignored', (err) => {
+exec('mkdir lib && babel --config-file ./package.babel.js ./src -d lib --extensions .js,.jsx,.snap --copy-files --no-copy-ignored', (err) => {
   if (err) {
     throw err
```

Comment:
btw, do we need this explicit throw?

Reply:
yes, because in other way we will not get any errors if something will go wrong with command execution